java/.gitignore

```diff
@@ -1 +1,3 @@
 *.class
```

Comment:
for Maven projects "target" should be ignored as well

Reply:
You are right. I replaced the existing `*.class` ignore by the entire `/target` directory.